students/Gabe_Lamberth/lesson02/grid_printer.py

```diff
@@ -0,0 +1,61 @@
+def plus(): 
+    return('+ ')
+
+def dash():
+    return('- ') 
+
+def bar():
+    return '| '
+
```

Comment:
probably don't need these functions.  Easier just to use a variable `plus = '+ '` as an example